packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -373,8 +379,10 @@ export default class TableResize implements EditorPlugin {
     ) {
         if (this.currentTd != td) {
             if (this.currentTd) {
-                this.resizerContainer.removeChild(this.horizontalResizer);
-                this.resizerContainer.removeChild(this.verticalResizer);
+                if (this.resizerContainer) {
+                    this.resizerContainer.removeChild(this.horizontalResizer);
```

Comment:
Since we don't have a stable repro, we don't even know if this.horizontalResizer exists, or if this.resizerContainer is the parent of this.horizontalResizer.

So I suggest to do
```
this.horizontalResizer?.parentNode?.removeChild(this.horizontalResizer);
```

Reply:
same for other places